docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMapping.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+
+namespace Samples.Dynamic
+{
+    public static class CustomMapping
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
+            var samples = new List<InputData>
+            {
+                new InputData { Age = 26 },
+                new InputData { Age = 35 },
+                new InputData { Age = 34 },
+                new InputData { Age = 28 },
+            };
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+
+            // We define the custom mapping between input and output rows that will be applied by the transformation.
+            Action<InputData, CustomMappingOutput > mapping =
+                (input, output) => output.IsUnderThirty = input.Age < 30;
+
+            // Custom transformations can be used to transform data directly, or as part of a pipeline of estimators.
+            // Note: If contractName is null in the CustomMapping estimator, any pipeline of estimators containing it, cannot be saved and loaded back. 
+            // See other sample on how to load and save the CustomMapping estimator.
```

Comment:
>// See other sample on how to load and save the CustomMapping estimator. [](start = 12, length = 72)

I would avoid referencing other samples from samples because it's hard to maintain.